.github/workflows/ci.yml

```diff
@@ -14,18 +14,21 @@ jobs:
 
       - name: Build docker image
         run: docker build . -t nominatim
-        working-directory: 3.6
+        working-directory: 3.7
```

Comment:
do you think we can build for several versions? otherwise we would need to manually build / update older versions if there is a change.

Reply:
I think that would be possible using a matrix build.

However, I'm snowed in with work in the next couple of weeks so if someone wants to take this to completion, please go ahead.